.babelrc

```diff
@@ -4,10 +4,20 @@
     ["env", {
       "targets": {
         "uglify": true,
-        "browsers": ["last 2 versions", "ie 11", "not ie 10"]
+        /* This is the full list of browsers we support, but we are not using it because we want babel-polyfill
+         * to include only polyfills for more modern browsers. 
+         * Older browser polyfills are in preModernBrowserPolyfills.js
+         */
+        /* "browsers": ["Chrome 60", "Firefox 57", "iOS 9", "Edge 14", "ChromeAndroid 64", "Safari 10", "ie 11"] */
```

Comment:
I looked in GA and found all the browsers that have greater that 0.5% usage over the last year.

Reply:
GA is probably the most definitive, but it's interesting that the [USA analytics](https://analytics.usa.gov/veterans-affairs/) for all Veteran Affairs sites shows IE7 at 7.9%.